admin-api/how-to-use.md

```diff
@@ -0,0 +1,280 @@
+    "description": "",
+    "enabled": true,
+    "lifetime": 3600,
+    "scopes": [
+        "api_client_read",
+        "api_client_write",
+        "cart_rule_write",
+        "customer_group_read",
+        "customer_group_write",
+        "hook_read",
+        "hook_write",
+        "module_read",
+        "module_write",
+        "product_read",
+        "product_write"
+    ]
+}
+```
+
+Congrats! You have successfully authenticated your client and retrieved information about it. You can now use the access token to access other endpoints in the API.
+
+Please note that the list of scopes may vary depending on the client you created. It is also important to note that the list of scopes here is the list of all scopes the client can use, not the list of scopes the client is using at the moment, with the generated access token.
+
+If you request an endpoint without the required scope, you will receive an error message. For example, if you try to access the `/admin-api/products` endpoint without the `product_read` scope in your access token, you will receive the following response:
+
+```json
+{
+    "type": "https://tools.ietf.org/html/rfc2616#section-10",
+    "title": "An error occurred",
+    "status": 403,
+    "detail": "Invalid scope.",
+    "class": "Symfony\\Component\\HttpKernel\\Exception\\AccessDeniedHttpException",
+    "trace": []
+}
+```
+
+## Retrieve a single resource
+
+Now that you have a better understanding of how to authenticate your client and retrieve information about it, you can try to retrieve other resources from the API. You can use the access token you generated earlier to authenticate your client.
+
+In the following example, we will try retrieving and saving user group information. To do that, you need to send a `GET` request to the `/admin-api/customers/group/{customerGroupId}` endpoint, where `{customerGroupId}` is the ID of the user group you want to retrieve. You need to provide the access token in the `Authorization` header.
+
+```bash
+curl --location 'http://yourdomain.test/admin-api/customers/group/1' \
+--header 'Authorization: Bearer YOUR_ACCESS_TOKEN'
+```
+
+If everything goes well, you should receive a response with information about the user group. The response should look like this:
+
+```json
+{
+    "customerGroupId": 1,
+    "localizedNames": {
+        "1": "Odwiedzający",
+        "2": "Visitor"
+    },
+    "reductionPercent": 0.0,
+    "displayPriceTaxExcluded": false,
+    "showPrice": true,
+    "shopIds": [
+        1
+    ]
+}
+```
+
+Notice that you can access the localized names of the user group and all other relevant information.
+
+If you access the endpoint with the wrong ID, you will receive an error message. For example, if you try to access the `/admin-api/customers/group/999999` endpoint, you will receive the following response:
+
+```json
+{
+    "type": "https://tools.ietf.org/html/rfc2616#section-10",
+    "title": "An error occurred",
+    "status": 500,
```

Comment:
```suggestion
    "status": 404,
```

Reply:
No, I believe the API returns this code, although the HTTP code itself is the right one. This is one of the things we need to improve but so far the doc matches the implementation When it's improved we can update this part as well